react/components/EXPERIMENTAL_Conditions/Atoms/ObjectAtom.js

```diff
@@ -11,6 +11,11 @@ class ObjectAtom extends React.Component {
     </div>
   )
 
+  handleOnChangeObjectCallback = (newValue, structure) => {
```

Comment:
Do you need this function? I think you can pass `this.props. onChangeObjectCallback` directly.